packages/core-mobile/package.json

```diff
@@ -65,7 +65,7 @@
     "@react-native-firebase/messaging": "21.0.0",
     "@react-native-google-signin/google-signin": "13.1.0",
     "@react-native-menu/menu": "1.1.3",
-    "@react-navigation/bottom-tabs": "6.6.1",
```

Comment:
I had to downgrade this to match the version used in expo-router. If we don't, it raises an error when this package is used.